src/sass/components/_top.scss

```diff
@@ -214,10 +214,45 @@
     }
 }
 
+.menu-children {
+    display: none;
+    
+    &.visible {
+        display: block;
+        position: absolute;
+        background: #f5f5f5;
+        width: 130px;
+        z-index: 99;
+        margin: 0 !important;
+        padding: 0 !important;
```

Comment:
lines 226 and 248 (both margin: 0) can be removed. If I remove anything else, the layout screws up.